style.css

```diff
@@ -208,3 +229,141 @@ h3 {
 .m:hover {
   cursor: pointer;
 }
+
+.work-container {
+  flex-direction: column;
+}
+
+h2 {
+  font-size: 40px;
+  font-weight: 500;
+  line-height: 52px;
+  text-align: left;
+  color: rgba(148, 144, 255, 1);
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.breakline-work {
+  margin: 33px 24px 33px 24px;
+}
+
+.work1 {
+  display: flex;
```

Comment:
**Optional Changes**
This is marked optional but you should consider changing it.
- [ ] The value `flex` for the css selector property `display` is repeated several times. You could refactor it into a `.flex` to make your code as `DRY` as possible. This will make your css file more cleaner.

Reply:
Thank you for your suggestions. However, I don't understand the point in doing so or how to do it.

Reply:
- [ ] [OPTIONAL] To perfectly apply the DRY concept. It's good to not repeat the same lines of code instead try to use classes. For example `display: flex;` is repeated a lot of times you can create class 
`.flex {
  display: flex;
} ` and place it in the HTML whenever you need it. 

DRY concept is one of coding  best practices .